Package/Installer.cs

```diff
@@ -251,8 +279,51 @@ private FileInfo[] GetFilesInUse(string tapDir, List<string> packagePaths)
             return filesInUse.ToArray();
         }
 
+        private Action postInstallHook = () => { };
+
+        private void WaitForPacakgeFilesFreeWindows(List<string> packagePaths)
+        {
+            var allfiles = packagePaths.SelectMany(PluginInstaller.FilesInPackage).ToArray();
+
+            retry:
+            var procs = RestartManager.GetProcessesUsingFiles(allfiles);
+            if (procs.Count == 0)
+                return;
+            var msg = new StringBuilder();
+            msg.AppendLine("The following applications are blocking the installation:");
```

Comment:
This is a bit counterintuitive when it happens during uninstall.

Maybe replace "installation with "The operation".